iphone/CoreApi/CoreApi/PlacePageData/Common/PlacePageInfoData.mm

```diff
@@ -83,6 +83,9 @@ - (instancetype)initWithRawData:(Info const &)rawData ohLocalization:(id<IOpenin
     
     _atm = rawData.HasAtm() ? NSLocalizedString(@"type.amenity.atm", nil) : nil;
       
+    std::string wheelchair = rawData.GetLocalizedWheelchairType();
```

Comment:
How editing of wheelchair accessibility will be supported? Usually, when we add some fields into Place Page, we assume that they are/will be editable.

Does it make sense to introduce and pass an enum from the C++ core, and localize it in the UI? Or just some hard-coded keys-strings "unknown", "yes", "no", and "limited"?

Also think about Android implementation to make it easier. Maybe something similar to already existing editor implementations like WiFi editing?